pkg/lib/config/environment.go

```diff
@@ -50,6 +50,10 @@ type EnvironmentConfig struct {
 	DevMode DevMode `envconfig:"DEV_MODE" default:"false"`
 	// LogLevel sets the global log level
 	LogLevel string `envconfig:"LOG_LEVEL" default:"warn"`
+	// HTTPProxy sets the HTTP proxy
+	HTTPProxy string `envconfig:"HTTP_PROXY"`
+	// SSLCertFile sets additional SSL certificate file
+	SSLCertFile string `envconfig:"SSL_CERT_FILE"`
```

Comment:
The name of these variables is too general, making people think Authgear will support them. But in fact, they are used in e2e test only. I suggest we make the authgear command to take 2 extra CLI options, `--e2e-http-proxy-url` and `--e2e-tls-ca-cert-path`. When the CLI options are given, then the HTTP client used by OAuth will be configured accordingly.